### DISCUSSION TITLE
E492: Not an editor command: CoCInfo

### DISCUSSION THREAD

**Discussion by putcut** [neoclide/coc.nvim#5111] [Q&A] [answered]
I am using LazyVim.
coc.nvim looks installed, `:checkhealth` looks OK, but at `:CoCInfo` I get `E492: Not an editor command`.  (same for `:CoCInstall`)

coc.lua is below and that is all I have added since setting up LazyVim.

```
return {
  {
    "neoclide/coc.nvim",
    breanch = "release",
  },
}
```

**Comment by fannheyward** [comment, marked as answer]
It's `:CocInfo`, the second `c` is lowercase.

**Reply by putcut** [reply]
Oops, sorry! Thank you!
